docs/guides/mfa/20_webauthn.mdx

```diff
@@ -1,6 +1,6 @@
 ---
 id: webauthn-fido-yubikey
-title: WebAuthn / FIDO2 (YubiKey)
+title: WebAuthn/FIDO2 (YubiKey)
```

Comment:
nit
I would prefer no "/" in headings, maybe just 
```suggestion
title: WebAuthn and FIDO2 (YubiKey)
```
?

Reply:
Yea, that's okay, we can do it :)